test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -1590,6 +1642,21 @@ func isHugePageInAvailable(rl corev1.ResourceList) bool {
 	return false
 }
 
+func isRequestingHugepages(podRes podResourcesRequest) bool {
+	for _, appContainer := range podRes.appCnt {
+		if isHugePageInAvailable(appContainer) {
```

Comment:
minor: in a future PR we should rename this function, like `isHugepagePresent` or so. This is a bit misleading in this current form

Reply:
were you referring to renaming isRequestingHugepages or isHugePageInAvailable? because isHugepagePresent sounds better name for isHugePageInAvailable

Reply:
I meant that we should move away from `isHugePageInAvailable`, which is no longer fit, and change to something like `isHugepagePresent`, or something else along these lines